modules/common.nix

```diff
@@ -8,10 +8,15 @@
   lib,
   ...
 }: let
+  inherit (lib.attrsets) mapAttrsToList;
+  inherit (lib.lists) isList;
+  inherit (lib.strings) concatStringsSep;
   inherit (lib.modules) mkIf mkDefault mkDerivedConfig;
   inherit (lib.options) mkOption mkEnableOption literalExpression;
   inherit (lib.strings) hasPrefix;
-  inherit (lib.types) bool submodule str path attrsOf nullOr lines listOf package;
+  inherit (lib.types) bool submodule str path attrsOf nullOr lines listOf package oneOf int;
+
+  inherit (pkgs) writeShellScript;
```

Comment:
```suggestion
```

I don't think this will be re-used. Referring to it with `pkgs.writeShellScript` is fine.